src/main/java/byteceps/validators/HelpValidator.java

```diff
@@ -0,0 +1,36 @@
+package byteceps.validators;
+
+
+import byteceps.errors.Exceptions;
+import byteceps.ui.strings.HelpStrings;
+import byteceps.commands.Parser;
+
+
+//@@author LWachtel1
+public class HelpValidator {
+
+
+    //@@author LWachtel1
+    //@@author joshualeejunyi
```

Comment:
I don't think this works? Reposense attributes the author until the next author tag if I'm not wrong.